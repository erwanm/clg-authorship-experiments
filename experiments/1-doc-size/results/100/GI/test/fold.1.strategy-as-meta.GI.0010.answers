1903-us-tjoas 1854-hbs-smofl	0.50897
1895-wta-alae 1906-mt-cfma	0.537929
1891-fmc-ttf 1881-haj-fcbtp	1
1872-haj-ptf 1906-rwc-ttolp	0.570897
1898-hj-ttm 1879-hj-h	0.5105
1874-wta-tcw 1900-haj-adoh	0.342184
1878-hj-te 1882-lma-ajsb-v6	0.361951
1883-hj-rh 1898-hj-itc	1
1885-hj-gr 1910-us-sts	0.434133
1914-us-sm 1897-ewaoc-tdof	0.286868
1896-espw-cfal 1887-espw-jtf	1
1887-espw-tgb 1900-espw-wtg	0.398229
1894-cdw-tgh 1907-fmc-tdr	0.349969
1900-fmc-itpotk 1905-fmc-fm	0.430018
1889-fmc-g 1898-haj-tyb	0.779229
1890-hj-ttm 1904-hj-tgb	0.682298
1878-hbs-pp 1870-hbs-tbc	0.246614
1901-us-km 1921-us-tbol	0.128572
1883-hj-rh-v2 1872-wta-lb	0
1866-haj-ccc-ia 1871-hbs-mwai	0.569783
1880-hj-c 1885-hj-altof	0.603859
1898-fmc-ari-v2 1893-fmc-kl-v2	0.266178
1918-rwc-tlg 1863-wta-isao	0.061414
1884-ab-sbts 1898-us-apom	0.324409
1889-cfw-jl 1915-rwc-p	0.535316
1872-mt-ri 1895-haj-tdw	0.181434
1902-fmc-c 1883-fmc-tl	0.524212
1859-hbs-tmw 1887-fmc-pp	0.973383
1884-wta-Sac 1882-wta-aa	1
1871-haj-tt 1876-wta-vao	0.726647
1913-es-tm 1911-rwc-tgr	0.457577
1905-es-tpoe 1907-es-tlotg	0.762707
1899-haj-mmv 1866-haj-hf	0.529474
1891-fmc-do 1876-cdw-itl	0.289248
1892-fmc-pg 1883-fmc-dc	1
1917-ab-bn 1922-us-tcmc	0.421859
1889-cdw-sths 1869-espw-mwag	0.614001
1902-hj-twotd 1896-hj-tsop	0.418657
1888-haj-bb 1899-haj-ra	0.213328
1913-ew-tcotc 1868-haj-rd	0.187596
1868-espw-tga 1919-rwc-tsos	0.924329
1900-haj-ainy 1919-ew-m	0
1881-mt-tpatp 1899-haj-aoatb	0.135176
1891-fmc-k 1893-fmc-tr	0.762897
1881-hj-tpoal 1890-fmc-acmr	0.562154
1869-haj-mtmb 1903-haj-bba-ph	0.227154
1885-fmc-atoalp 1905-fmc-wso	1
1868-wta-mob 1870-wta-faf	0.40151
1871-hbs-pawt 1904-rwc-isotu	0.541628
1875-cfw-cn 1864-lma-m	0.874497
1914-mh+-mhce-nf 1908-fmc-tp	0.486153
1893-fmc-md 1895-fmc-ajs	1
1864-wta-tsb 1883-wta-sh	0.976243
1900-rwc-taoamm 1910-rwc-tgm	0.609441
1903-fmc-thor 1887-fmc-s	0.969966
1884-mt-taohf 1874-mt-lotm	1
1907-espw-tldup 1871-espw-tsp	0.745643
1866-espw-gcj 1882-haj-bn	0.445523
1883-haj-tbb 1888-es-ttatv	0.903417
1908-fmc-s 1908-fmc-tws	1
1889-wta-wtel 1911-rwc-tcl	0.299666
1904-rwc-sm 1886-hj-pc	0.751635
1918-us-jh 1897-ew-tdof	0.837518
1872-lma-ss 1867-lma-tagomtt	0.135051
1910-ew-tomag 1889-es-atg	1
1889-espw-tsfi 1895-espw-tsojc	0.62159
1905-haj-fftf 1870-haj-rar	0.436525
1882-espw-dz 1909-mt-isd	0.328678
1907-ew-tfott 1896-hj-toh	1
1868-wta-dad 1885-wta-sts	0.560069
1909-rwc-tdm 1897-ab-tdohy	1
1891-haj-tetb 1868-haj-faf	0.714082
1893-haj-dtn 1866-haj-tcw	0.907644
1869-wta-dtr 1865-wta-waw	0.716336
1891-cdw-oi 1873-hbs-wish	0.780493
1879-hj-dm 1903-hj-ta	0.580371
1874-cfw-tac 1882-fmc-mi	0.650189
1893-espw-dm 1896-ab-boat	0.344672
1896-espw-tsasa 1894-espw-asl	0.538213
1879-haj-tym 1869-haj-pjb	0.600971
1920-us-1tsoap 1913-us-dg	0.600298
1917-es-ow-sh 1873-hbs-pl	0.752573
1855-wta-aa 1873-wta-tyc	0
1879-haj-ttb 1894-fmc-wg	0
1898-fmc-vc 1895-fmc-t	0.70918
1867-espw-gyatgc 1901-espw-a	0.455808
1896-fmc-c 1885-fmc-zamc	0.811753
1855-hbs-tmf 1866-espw-gb	1
1897-fmc-aroy 1884-fmc-aap	1
1883-espw-btg 1911-hj-to	0.155124
1862-hbs-tpooi 1875-hbs-waon	1
1883-fmc-ars 1885-fmc-tcotk	0.545811
1886-hbs-tsw 1854-hbs-smofl-v2	0.188571
1903-ew-ivatg 1878-cdw-hikab	0.45578
1858-wta-pap 1897-haj-fof	0.444269
1902-hj-twotd-v2 1888-hj-tr	0.619683
1871-hbs-ofs 1888-fmc-si	0.555247
1907-ab-rm 1901-hj-tsf	0.097165
1882-haj-ffbts 1893-haj-ianwatgfoa	0
1908-rwc-tfl 1910-rwc-ap	1
