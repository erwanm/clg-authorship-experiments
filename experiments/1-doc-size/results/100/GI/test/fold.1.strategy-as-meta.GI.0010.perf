0.364440	0.607400	0.600000
