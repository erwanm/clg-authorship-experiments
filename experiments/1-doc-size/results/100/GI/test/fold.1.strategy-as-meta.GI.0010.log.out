apply-config.sh: applying config '/tmp/100/process/selected-meta-configs/fold.1.strategy-as-meta.GI.0010' to cases '/home/moreaue/dump/expe1//results/100/GI/test/only-cases.tmp.1mTmtWbG'; results written to '/home/moreaue/dump/expe1//results/100/GI/test/fold.1.strategy-as-meta.GI.0010'
apply-config.sh: done.
